docs/quick-start.md

```diff
@@ -120,53 +121,82 @@ package main
 
 import (
 	"fmt"
+	"log"
+
 	"github.com/rubrikinc/rubrik-sdk-for-go/rubrikcdm"
 )
 
 func main() {
+
 	// Establish a connection to the Rubrik cluster
-	rubrik := rubrikcdm.ConnectEnv()
+	rubrik, err := rubrikcdm.ConnectEnv()
```

Comment:
I get an error on this - `assignment mismatch: 2 variables but 1 values`

Reply:
@chriswahl Did you upgrade your code to the `devel` branch? That error is consistent with how everything looked before the new updates.

Reply:
I'm using the `shamsway:master` branch.

Reply:
@chriswahl there isn't a quick start in the `devel` brach, so I forked the master branch to make updates. It makes sense to me to have the quick start in the `devel` branch as well, so we will do that moving forward.